### PR TITLE
Fix for CVE-2014-3566 'POODLE' SSL attack

### PR DESCRIPTION
**Pull request by briandrawert**
original commit '22c4922f4796078c5613de9e07e66b8764549cad'

### CODE REVIEW

File: IPython/html/notebookapp.py [hunk 1]
```diff
@@ -832,6 +833,8 @@ def init_webapp(self):
                 ssl_options['keyfile'] = self.keyfile
         else:
             ssl_options = None
+        # Disable SSLv3, since its use is discouraged.
+        ssl_options['ssl_version']=ssl.PROTOCOL_TLSv1
```
Comment (minrk) [inline IPython/html/notebookapp.py]: This will fail when ssl_options is None. It should be moved into the `if` block above.

**Comment by minrk** [comment]
The 'merged with master' commit shouldn't exist. Can you remove it?

**Comment by briandrawert** [comment]
The commit was removed, but now the PR has conflicts.  This is because I was developing off of the 3.0.0 tag, not HEAD.  The 'merged with master' commit was the result of pulling from master and resolving the conflicts.

**Comment by minrk** [comment]
Ah, in that case, can you rebase on master?

**Comment by briandrawert** [comment]
This is deprecated by: https://github.com/ipython/ipython/pull/8179

**Comment by minrk** [comment]
You don't need to open a new PR for a rebase, but that's fine. #8179 has been merged and backported. Thanks!

**Pull request closed without merging**